client_app/src/components/shelter/ShiftDetails.js

```diff
@@ -3,52 +3,80 @@ import "../../styles/shelter/ShiftDetails.css";
-    }
+    const handleChange = (event) => {
+      setSelectedShift(event.target.value);
+    };
 
     const onSignUpVolunteersClick = () => {
-      setIsMoreDetailsModelOpen(false);
       setIsVolunteerModalOpen(true);
     }
 
-    const currentTime = dayjs();
-    const startHour = currentTime.add(1, 'hour').startOf('hour');
-    const endHour = startHour.add(1, 'hour');
+    const onSendEmergencyAlertClick = () => {
+      setIsEmergencyModalOpen(true);
+    }
+
+    const onStatusModalClick = shift => {
+      setShift(shift);
+      setIsStatusModalOpen(true);
+    }
+
     return (
       <div>
-        {isMoreDetailsModelOpen} < MoreDetailsBox onSignUpVolunteersClick={onSignUpVolunteersClick} isMoreDetailsModelOpen={isMoreDetailsModelOpen} setIsMoreDetailsModelOpen={setIsMoreDetailsModelOpen} isVolunteerModalOpen={isVolunteerModalOpen}/>
-        {isVolunteerModalOpen} < ShiftsModal isVolunteerModalOpen={isVolunteerModalOpen} setIsVolunteerModalOpen={setIsVolunteerModalOpen} />
+        {isVolunteerModalOpen && <ShiftsModal isVolunteerModalOpen={isVolunteerModalOpen} setIsVolunteerModalOpen={setIsVolunteerModalOpen} />}
+        {isEmergencyModalOpen && <EmergencyAlertModal isEmergencyModalOpen={isEmergencyModalOpen} setIsEmergencyModalOpen={setIsEmergencyModalOpen} />}
+        {isStatusModalOpen && <EditStatusConfirmationModal isStatusModalOpen={isStatusModalOpen} setIsStatusModalOpen={setIsStatusModalOpen} shift={shift}/>}
         <div className="shift-details">
           <div className="datetime-picker">
-            <h4 className="date-label">Date</h4>
             <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <h4 className="date-label">Start Date: </h4>
+              <DatePicker defaultValue={dayjs()} slotProps={{textField: {size: "small"}}} sx={{width: 200}} />
+              <h4 className="date-label">End Date: </h4>
               <DatePicker defaultValue={dayjs()} slotProps={{textField: {size: "small"}}} sx={{width: 200}} />
-              <h4 className="starttime-label">Shift Start Time</h4>
-              <TimePicker defaultValue={startHour}  slotProps={{textField: {size: "small"}}} sx={{width: 200}} />
-              <h4 className="endtime-label">Shift End Time</h4>
-              <TimePicker defaultValue={endHour} slotProps={{textField: {size: "small"}}} sx={{width: 200}} />
             </LocalizationProvider>
+            <h4 className="endtime-label">Select Shift(s): </h4>
+            <Box sx={{ minWidth: 120 }}>
+              <FormControl fullWidth>
+                <InputLabel id="demo-simple-select-label">Shift(s)</InputLabel>
+                <Select
+                  labelId="demo-simple-select-label"
+                  id="demo-simple-select"
+                  value={selectedShifts}
+                  label="Shift(s)"
+                  onChange={handleChange}
+                >
+                  <MenuItem value={"10:00 - 12:00"}>10:00 - 12:00</MenuItem>
```

Comment:
Should we include the shiftName (Morning, Afternoon,..) like in the wireframe?

Reply:
I was planning on it I will I just didn't add it yet